assets/js/pages/SapSystemsOverviewPage/SapSystemsOverview.jsx

```diff
@@ -77,6 +77,22 @@ function SapSystemsOverview({
         title: 'Tenant',
         key: 'tenant',
       },
+      {
+        title: 'Type',
+        key: 'type',
+        render: (_, items) =>
+          uniq(
+            flatMap(
+              filter(items.applicationInstances, { sap_system_id: items.id }),
```

Comment:
You shouldn't need this filter, as `applicationInstances` are already coming filtered.